src/libexpr/nixexpr.hh

```diff
@@ -252,16 +269,39 @@ struct ExprAttrs : Expr
     bool recursive;
     PosIdx pos;
     struct AttrDef {
-        bool inherited;
+        enum class Kind {
+            plain,
+            inherited,
+            inheritedFrom,
```

Comment:
```suggestion
            /** `attr = expr;` */
            Plain,
            /** `inherit attr1 attrn;` */
            Inherited,
            /** `inherit (expr) attr1 attrn;` */
            InheritedFrom,
```

Upper camel case, to follow the other `enum class`/`enum struct`s, except, unfortunately `LogFormat`.